src/main/java/it/gov/pagopa/ranking/dto/PageRankingDTO.java

```diff
@@ -0,0 +1,49 @@
+package it.gov.pagopa.ranking.dto;
+
+import lombok.*;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+@Getter
+@Setter
+public class PageRankingDTO<T> extends PageImpl<T> {
```

Comment:
Are we sure that we want to extend this model? If we want to use custom names and a subset of its fields, let's just define there here and not extend PageImpl anymore